apps/dapp-console/app/components/Tile/Tile.tsx

```diff
@@ -5,32 +5,73 @@ import {
   CardHeader,
 } from '@eth-optimism/ui-components/src/components/ui/card'
 import { Text } from '@eth-optimism/ui-components/src/components/ui/text'
+import { cn } from '@eth-optimism/ui-components/src/lib/utils'
```

Comment:
nit: do you think we should give this a different name? hard to tell what cn means

Reply:
this is directly from shadcn so i'd rather not change the name to keep consistency. `cn` here is referring to classname just abbreviated.